samples/src/main/java/com/example/bigquery/ExtractTableToJSON.java

```diff
@@ -0,0 +1,29 @@
+package com.example.bigquery;
+
+// [START bigquery_extract_table]
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.Table;
+
+public class ExtractTableToJSON {
+
+  public static void runExtractTableToJSON() {
+    // TODO(developer): Replace these variables before running the sample.
+    Table table = null;
+    String format = "CSV";
+    String bucketName = "my-bucket";
+    String gcsFileName = "gs://" + bucketName + "/extractTest.csv";
+    extractTableToJSON(table, format, gcsFileName);
+  }
+
+  // Exports my-dataset-name:my_table to gcs://my-bucket/my-file as raw CSV
+  public static void extractTableToJSON(Table table, String format, String gcsFileName) {
+
+    try {
+      table.extract(format, gcsFileName);
+      System.out.println("Table extraction job completed successfully");
```

Comment:
This should show the user how to interact with the value provided by `table.extract`.

Reply:
I made a change to show that value provided by `table.extract` is Job. They should be expecting to see extracted CSV in GCS bucket.